src/main/java/io/axoniq/axonserver/connector/impl/ControlChannelImpl.java

```diff
@@ -164,18 +165,20 @@ void handleReconnectRequest(PlatformOutboundInstruction platformOutboundInstruct
     public synchronized void connect() {
         StreamObserver<PlatformInboundInstruction> existing = instructionDispatcher.get();
         if (existing != null) {
-            logger.info("Not connecting - connection already present");
+            logger.info("ControlChannel for context '{}' is already connected", context);
         } else {
             PlatformOutboundInstructionHandler responseObserver =
-                    new PlatformOutboundInstructionHandler(clientIdentification.getClientId(), this::handleDisconnect);
-            logger.debug("Opening instruction stream");
+                    new PlatformOutboundInstructionHandler(clientIdentification.getClientId(), this::handleDisconnect,
+                                                           upstream -> {
```

Comment:
Why use a method `registerOutboundStream` in the `CommandChannelImpl` and a lambda in this class?